example/src/release_tests/swipeableReanimation/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { Text, Animated, StyleSheet, View } from 'react-native';
+
+import {
+  GestureHandlerRootView,
+  RectButton,
+} from 'react-native-gesture-handler';
+
+import { Swipeable } from 'react-native-gesture-handler';
+import ReanimatedSwipeable from 'react-native-gesture-handler/ReanimatedSwipeable';
+import Reanimated, {
+  SharedValue,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+
+function LeftAction(prog: SharedValue<number>, drag: SharedValue<number>) {
+  const styleAnimation = useAnimatedStyle(() => {
+    console.log('[R] showLeftProgress:', prog.value);
+    console.log('[R] appliedTranslation:', drag.value);
+
+    return {
+      transform: [{ translateX: drag.value - 50 }],
+    };
+  });
+
+  return (
+    <Reanimated.View style={styleAnimation}>
+      <Text style={styles.leftAction}>Text</Text>
+    </Reanimated.View>
+  );
+}
+
+function RightAction(prog: SharedValue<number>, drag: SharedValue<number>) {
+  const styleAnimation = useAnimatedStyle(() => {
+    console.log('[R] showRightProgress:', prog.value);
+    console.log('[R] appliedTranslation:', drag.value);
+
+    return {
+      transform: [{ translateX: drag.value + 50 }],
+    };
+  });
+
+  return (
+    <Reanimated.View style={styleAnimation}>
+      <Text style={styles.rightAction}>Text</Text>
+    </Reanimated.View>
+  );
+}
+
+function LegacyLeftAction(prog: any, drag: any) {
+  prog.addListener((value: any) => {
+    console.log('[L] showLeftProgress:', value.value);
+  });
+  drag.addListener((value: any) => {
+    console.log('[L] appliedTranslation:', value.value);
+  });
+
+  const trans = Animated.subtract(drag, 50);
+
+  return (
+    <RectButton>
```

Comment:
It would be nice to have some feedback if we use `RectButton` (could be for example `console.log`). Also, why legacy version uses `button` if the one based on `Reanimated` doesn't?